app/code/Magento/Theme/Controller/Result/AsyncCssPlugin.php

```diff
@@ -16,7 +16,7 @@
  */
 class AsyncCssPlugin
 {
-    private const XML_PATH_USE_CSS_CRITICAL_PATH = 'dev/css/use_css_critical_path';
+    const XML_PATH_USE_CSS_CRITICAL_PATH = 'dev/css/use_css_critical_path';
```

Comment:
Please revert changing constant visibility. As I understood - you needed it in tests, so please just create own const inside test class and use it there

Reply:
Yeah, sometimes failing integration tests like this, as requested I have reverted the constant change & added new stub in test class. can you please review now.

Reply:
Thanks for the feedback